tensorflow/python/data/experimental/service/server_lib.py

```diff
@@ -38,7 +38,7 @@ class DispatcherConfig(
     port: Specifies the port to bind to. A value of 0 indicates that the server
       may bind to any available port.
     protocol: The protocol to use for communicating with the tf.data service.
-      Acceptable values include `"grpc" and "grpc+local"`.
+      Defaults to the only acceptable value of `"grpc"`.
```

Comment:
How about removing this sentence entirely? Users can register arbitrary protocols by dynamically linking them into the tensorflow binary, so there could be more acceptable values.

Reply:
just mentioned it as "Defaults to grpc".